organizing_hub/tasks.py

```diff
@@ -238,6 +239,9 @@ def build_contact_list_for_event_promotion(event_promotion_id):
     event_promotion_id : int
         EventPromotion id
 
+    cons_group : int
+        BSD constituent group id
```

Comment:
there is no param in this method for cons group?

Reply:
Agh wrong method.